lib/permutations.rb

```diff
@@ -1,4 +1,37 @@
+# Write a method which will take two strings as arguments 
+# and returns true if one string is a permutation of the other.
 
 def permutations?(string1, string2)
```

Comment:
👍 This works!

Just a note, you're repeating things a few times here, maybe a helper method?